exercises/practice/leap/leap_test.sql

```diff
@@ -0,0 +1,59 @@
+-- Create database:
+.read ./create_fixture.sql
+
+-- Read user student solution and save any output as markdown in user_output.md:
+.mode markdown
+.output user_output.md
+.read ./success.sql
+.output
+
+-- Create a clean testing environment:
+DROP TABLE IF EXISTS main.tests;
+CREATE TABLE IF NOT EXISTS main.tests (
```

Comment:
The `tests` table is sometimes referred to with a namespace and other times without. Should that be consistent?